plugins/service-monitor-activemq/src/main/java/com/thinkbiganalytics/service/activemq/config/ActivemqServiceCheckSpringConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.thinkbiganalytics.service.activemq.config;
+
+
+/*-
+ * #%L
+ * thinkbig-service-monitor-nifi
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)